testing/vulkan-headers/APKBUILD

```diff
@@ -0,0 +1,75 @@
+# Contributor: Mark Riedesel <mark@klowner.com>
+# Maintainer: Mark Riedesel <mark@klowner.com>
+pkgbase=vulkan
+pkgname=$pkgbase-headers
+pkgver=1.0.34
+pkgrel=0
+pkgdesc="Vulkan development headers"
+url="https://www.khronos.org/vulkan/"
+arch="noarch"
+license="custom"
+depends=""
+makedepends="asciidoc tinyxml2-dev gcc sed python3"
+install=""
+subpackages="$pkgbase-doc $pkgbase-html-doc:_htmldoc"
+archive_name="Vulkan-Docs"
+source="$pkgname-$pkgver.tar.gz::https://github.com/KhronosGroup/Vulkan-Docs/archive/v${pkgver}-core.tar.gz
+	https://raw.githubusercontent.com/KhronosGroup/Vulkan-Hpp/master/VulkanHppGenerator.cpp"
```

Comment:
This seems to download git master version of the file, which is since changed. Please fix this by one:
1. Package Vulkan-Hpp separately, and makedepend on it (preferred)
1. Add a Vulkan-Hpp release tarball as dependency here, and use files from it
1. Using download url including git hash, in that case the downloaded filename should be versioned too